Casks/sonixd.rb

```diff
@@ -0,0 +1,36 @@
+# typed: false
+# frozen_string_literal: true
+
+cask "sonixd" do
```

Comment:
```suggestion
cask "sonixd" do
```
Please remove the lines from the start of the file.